aspnetcore/test/load-tests.md

```diff
@@ -0,0 +1,76 @@
+---
+title: ASP.NET Core load/stress testing
+author: Jeremy-Meng
+description: Describes several notable tools and approaches for load testing and stress testing ASP.NET Core apps.
+ms.author: riande
+ms.custom: mvc
+ms.date: 01/04/2019
+uid: test/loadtests
+---
+# Load and stress testing ASP.NET Core
+
+Load testing and stress testing are important to ensure a web app is performant and scalable. Their goals are different even they often share similar tests.
+
+**Load tests**: Tests whether the app can handle a specified load of users for a certain scenario while still satisfying the response goal. The app is run under normal conditions.
+
+**Stress tests**: Tests app stability when running under extreme conditions and often a long period of time:
+
+* High user load – either spikes or gradually increasing.
+* Limited computing resources.  
+
+Under stress, can the app recover from failure and gracefully return to expected behavior? The app is run under normal conditions.
```

Comment:
Is this supposed to say "The app is run under normal conditions" or is that a copy/paste error from the **Load tests** part. It sounds like stress test conditions are *not* normal.

Reply:
@Rick-Anderson yes, under stress test, the app is running under unusual conditions.